apps/asap-server/src/entities/news-and-events.ts

```diff
@@ -12,7 +12,9 @@ export const parseNewsAndEvents = (
     text: item.data.text?.iv,
     link: item.data.link?.iv,
     linkText: item.data.linkText?.iv,
-    thumbnail: item.data.thumbnail && createURL(item.data.thumbnail?.iv)[0],
+    thumbnail:
+      (item.data.thumbnail?.iv && createURL(item.data.thumbnail.iv)[0]) ||
+      undefined,
     title: item.data.title.iv,
     type: item.data.type.iv,
   } as NewsOrEventResponse);
```

Comment:
Should this be casted?

Reply:
i dont see anything here being casted? unless you would like me to? anyway this is WIP, you guys act fast. I am now changing the generic Rest type to reflect it

Reply:
as something casts. Ideally it be const blah: NewsOrEventResponse = { ....}

Reply:
yeah i know what a type cast is, somehow i didn't see the `as` in line 18. 

Definitely a valid request, i dropped the cast as it wasn't even necessary:
https://github.com/yldio/asap-hub/pull/815/commits/a95fd7d40153c95c742f169e0cc60490df8139f1